css/structure/jquery.mobile.transition.fade.css

```diff
@@ -1,27 +1,39 @@
 @-webkit-keyframes fadein {
-    from { opacity: 0; }
-    to { opacity: 1; }
+	from { opacity: 0;
```

Comment:
This rule and the ones below should really be indented as
```CSS
from {
    opacity: 0;
}
```
if writing them on one line is no longer considered OK.